build/build_config.h

```diff
@@ -113,7 +113,7 @@
 #define ARCH_CPU_ARMEL 1
 #define ARCH_CPU_32_BITS 1
 #define ARCH_CPU_LITTLE_ENDIAN 1
-#elif defined(__aarch64__)
+#elif defined(__aarch64__) || defined(_M_ARM64)
```

Comment:
I'm really surprised just `__aarch64__` for Clang on Windows.

Reply:
Well, we still support non-clang local builds, at least in theory.